src/normal_form_game.jl

```diff
@@ -691,3 +691,107 @@ Return true if `action_profile` is Pareto dominant for game `g`.
 
 * `::Bool`
 """ is_pareto_dominant
+
+# is_dominated
+
+"""
+    is_dominated(player, action[, tol=1e-8, lp_solver=ClpSolver()])
+
+Determine whether `action` is strictly dominated by some mixed
+action.
+
+# Arguments
+
+- `player::Player` : Player instance.
+- `action::PureAction` : Integer representing a pure action.
+- `tol::Real` : Tolerance to be used.
+- `lp_solver::AbstractMathProgSolver` : Allows users to choose a particular
+  solver for linear programming problems. Options include ClpSolver(),
+  CbcSolver(), GLPKSolverLP() and GurobiSolver(). By default, it choooses
+  ClpSolver().
+
+# Returns
+
+- `::Bool` : True if `action` is strictly dominated by some mixed action;
+  False otherwise.
+
+"""
+function is_dominated(player::Player{N,T}, action::PureAction;
+                      tol::Real=1e-8,
+                      lp_solver::MathProgBase.AbstractMathProgSolver=
+                      ClpSolver()) where {N,T<:Real}
+    payoff_array = player.payoff_array
+    S = typeof(zero(T)/one(T))
+
+    m, n = size(payoff_array, 1) - 1, prod(size(player.payoff_array)[2:end])
+
+    ind = trues(num_actions(player))
+    ind[action] = false
+
+    A = Array{S}(undef, n+1, m+1)
+    A[1:n, 1:m] = transpose(reshape(-selectdim(payoff_array, 1, ind) .+
+                                    selectdim(payoff_array, 1, action:action),
+                                    (m, n)))
+    A[1:n, m+1] .= 1
+    A[n+1, 1:m] .= 1
+    A[n+1, m+1] = 0
+
+    b = zeros(S, n+1)
+    b[end] = 1
+
+    c = zeros(S, m+1)
+    c[end] = -1
+
+    sense = Array{Char}(undef, n+1)
+    sense[1:n] .= '<'
```

Comment:
Here I got

```
┌ Warning: broadcast will default to iterating over its arguments in the future. Wrap arguments of
│ type `x::Char` with `Ref(x)` to ensure they broadcast as "scalar" elements.
│   caller = ip:0x0
└ @ Core :-1
```

The warning says you should do

```jl
sense[1:n] .= Ref('<')
```

though I don't know what `Ref` means.

Alternatively you can just write

```jl
for i in 1:n
    sense[i] = '<'
end
```